zeppelin/hdi_add_zeppelin.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+#----------------------------------------------------------------
+# THIS SCRIPT ADDS INSTALLS AND STARTS ZEPPELIN ON HBASE CLUSTER 
+#----------------------------------------------------------------
+
+
+#----------------------------------------------------------------
+# PRINT USAGE INFORMATION
+#----------------------------------------------------------------
+
+
+print_usage()
+{
+cat << ...
+Usage:
+$0
+
+Mandatory arguments:
+--------------------
+
+-p
+  Admin password for Ambari
+  
+-c
+  DNS name of HBase Cluster
+-h
+  FQDN of head node host
```

Comment:
You can also figure this out from cluster manifest. It should be wherever the gateway points to.

Reply:
corrected message, will take it from user